sdk/metricsadvisor/tests.yml

```diff
@@ -10,3 +10,23 @@ jobs:
           safeName: azureaimetricsadvisor
       EnvVars:
         AZURE_TEST_MODE: RECORD
+        AZURE_METRICS_ADVISOR_ENDPOINT: $(metricsadvisor-test-service-endpoint)
```

Comment:
Are these static test configurations or dynamic each time the tests are ran?

Reply:
They are static since the service has a restriction on resources per region and also the tests need ingested data which requires some prior preparation.